src/components/Modal/BaseModal.js

```diff
@@ -30,17 +30,26 @@ class BaseModal extends PureComponent {
         this.hideModal = this.hideModal.bind(this);
     }
 
+    componentDidMount() {
+        if (!this.props.isVisible) { return; }
+        Modal.setModalClose(this.props.onClose);
+    }
```

Comment:
Is this necessary? I think modals are mounted when before they are visible, but maybe this is just to make sure? If so please add a comment explaining why.

Reply:
This is a special case for PopoverReportActionContextMenu.
I confirmed that `isVisible` is already true when component mount, so `componentDidUpdate` is never called.
![image](https://user-images.githubusercontent.com/108292595/221004980-45f8fea9-a960-48f4-bb40-8c2317a1e329.png)